src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/expressions/brgemm_copy_b_buffer_expressions.cpp

```diff
@@ -79,12 +78,12 @@ void CompensationsBufferExpression::init_allocation_size(
     [[maybe_unused]] const std::shared_ptr<snippets::lowered::LoopManager>& loop_manager,
     [[maybe_unused]] size_t allocation_rank) {
     const auto& parent_expr = get_input_port_connector(0)->get_source().get_expr();
+    const auto& brgemm_config = ov::as_type_ptr<ov::intel_cpu::BrgemmCopyB>(parent_expr->get_node())->get_config();
```

Comment:
Should we check that copyB is not nullptr after casting? At least to avoid static code analyzer warnings 😄